### PR TITLE
[MIG] crm_claim_code: Migration to 12.0

### PR DESCRIPTION
**Pull request by elrodo1**

### CODE REVIEW

**Review by luistorresm — CHANGES_REQUESTED** [review]
@elrodo1 

Please remove the file `crm_claim_code/.DS_Store`

Regards

File: crm_claim_code/__init__.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        # the field was already existing before the installation of the addon
+        return
+    with Environment.manage():
+        env = Environment(cr, SUPERUSER_ID, {})
+
+        sequence_model = env['ir.sequence']
+
+        claims = env['crm.claim'].search([], order="id")
+        for claim in claims:
+            claim.code = sequence_model.next_by_code('crm.claim')
```
Comment (hugho-ad) [inline crm_claim_code/__init__.py]: please, move these functions to a new hooks.py file then just import the names of the functions at __init__.py
like the following
`from .hooks import post_init_hook`

File: crm_claim_code/tests/test_crm_claim_code.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo.tests import common
+
+
+class TestCrmClaimCode(common.SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestCrmClaimCode, cls).setUpClass()
+        cls.crm_claim_model = cls.env['crm.claim']
```
Comment (hugho-ad) [inline crm_claim_code/tests/test_crm_claim_code.py:13]: Dont use built-in variables
cls is a python reserved variable

Comment (hugho-ad) [reply crm_claim_code/tests/test_crm_claim_code.py:13]: for get it, I got confuse

File: crm_claim_code/i18n/es.po [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+"Language: es\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#. module: crm_claim_code
+#: model:ir.model,name:crm_claim_code.model_crm_claim
+msgid "Claim"
+msgstr "Reclamación"
```
Comment (luisiniguezh) [inline crm_claim_code/i18n/es.po]: I think `Reclamo` would be a better translation

File: crm_claim_code/i18n/es.po [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+#. module: crm_claim_code
+#: model:ir.model.fields,field_description:crm_claim_code.field_crm_claim_code
+msgid "Claim Number"
+msgstr "Número de reclamación"
```
Comment (luisiniguezh) [inline crm_claim_code/i18n/es.po]: *`Número de reclamo`

**Review by max3903 — APPROVED** [review]

**Review by hugho-ad — APPROVED** [review]

**Pull request merged**